common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/model/vrang/VraNgBlueprint.java

```diff
@@ -67,4 +132,21 @@ public boolean equals(Object obj) {
 		VraNgBlueprint other = (VraNgBlueprint) obj;
 		return this.id.equals(other.getId());
 	}
+
+	/**
+	 * Using `id` should be enough here, as `id`s in Aria are unique. Adding more
+	 * details for extra uniquieness.
+	 *
+	 * @return the hashcode numerical repesentation of the object
+	 */
+	@Override
+	public int hashCode() {
+		return new HashCodeBuilder(PRIME_NUMBER_17, PRIME_NUMBER_31)
```

Comment:
Just for my info, why 17 and 31?

Reply:
I needed 2 random prime numbers, 😄  It's completely random 😄